cosmos/sql/tools/java/ycsb/recipes/README.md

```diff
@@ -58,25 +82,30 @@ A read recipe with a small read workload to familiarize you with the framework.
    - aggregation.csv has the aggregated results from all clients
     
      ![image](../../../../../../images/results-csv.png)
+   
+   - There will be a folder per VM with the detailed system diagnostics logs. These logs will help you diagnose issues. Check [common errors](#common-errors) section for details on errors.
+
+     ![image](../../../../../../images/results-diagnostics.png)
+
 
-6. Alternatively, create a paratemetr file or use the provided [sample parameter file](./parameter-files) to execute the recipe using [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli). Be sure to populate the parameter values in the parameter file.
-    -  Local Template
-     ```
-     az deployment group create \
-       --name <deploymen-name> \
-       --resource-group <resource-group-name> \
-       --template-file azuredeploy.json  \
-       --parameters parameter.json  
-      ```
-    - Remote Template
-    ```
-    az deployment group create --name <deploymen-name> \
-      --resource-group <resource-group-name> \
-      --template-uri "https://raw.githubusercontent.com/Azure/azure-db-benchmarking/main/cosmos/sql/tools/java/ycsb/recipes/read/try-it-read/azuredeploy.json" \
-      --parameters parameter.json
-    ```
 7. re-executing the recipe by setting "Skip Load Phase" to "true" , while leaving the rest of the parameter values unchanged, will execute just the read phase of the workload again, using the VM from the previous execution. 
 
+## Common Errors
+Following are the most common user mistakes that lead to errors. The error logs will be available in a container within the storage account provided. The only exception being the first error listed below. A unreachable storage account. In which case, the longs will be available only in the VM.
+
+1. Following error will appear in "agent.out" in the "/home/benchmarking" of the client VM, if a incorrect storage connecting is passed. 
```

Comment:
Do these errors log in agent.err as well? It's easier to read err file in comparison to out

Reply:
Application (YCSB) errors are not showing up in the error log, only script errors are showing up.

Reply:
made the change for storage error log.